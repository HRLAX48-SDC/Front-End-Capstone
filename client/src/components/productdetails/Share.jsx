import React from 'react';

function Share() {
  return (
    <div>
      <span>Share: </span>
      <a className='p_share_icon' href='https://www.facebook.com/'>
        <i className='fab fa-facebook-square fa-lg'></i>
      </a>
      <a className='p_share_icon' href='https://twitter.com/home'>
        <i className='fab fa-twitter-square fa-lg'></i>
      </a>
      <a className='p_share_icon' href='https://www.pinterest.com/'>
        <i className='fab fa-pinterest-square fa-lg'></i>
      </a>
    </div>
  )
}

export default Share;