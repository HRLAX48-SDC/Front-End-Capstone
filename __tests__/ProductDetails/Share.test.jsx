import React from 'react';
import renderer from 'react-test-renderer';
import Share from '../../client/src/components/productdetails/Share.jsx';

test('renders share component correctly', () => {
  const share = renderer.create(<Share />).toJSON();
  expect(share).toMatchSnapshot();
});
