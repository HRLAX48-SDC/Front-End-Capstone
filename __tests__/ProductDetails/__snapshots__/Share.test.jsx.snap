// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders share component correctly 1`] = `
<div>
  <a
    className="p_share_icon"
    href="https://www.facebook.com/"
  >
    <i
      className="fab fa-facebook-square fa-lg"
    />
  </a>
  <a
    className="p_share_icon"
    href="https://twitter.com/home"
  >
    <i
      className="fab fa-twitter-square fa-lg"
    />
  </a>
  <a
    className="p_share_icon"
    href="https://www.pinterest.com/"
  >
    <i
      className="fab fa-pinterest-square fa-lg"
    />
  </a>
</div>
`;
